etp-backend/src/main/clj/solita/etp/api/statistics.clj

```diff
@@ -5,18 +5,18 @@
             [solita.etp.service.statistics :as statistics-service]
             [solita.etp.schema.common :as common-schema]))
 
-(def routes
+(defn routes [use-cache?]
```

Comment:
Näitä ei mielestäni kannata alkaa säätää erikseen. Tässä on kaksivaihtoehtoa:
a) annetaan nämä vain olla; nämähän on periaatteessa ihan oikein koska tieto on julkista. Nythän nämä pitäisi toimia ihan järkevästi eli jää selaimen kakkuun
b) koko privaatti apiin laitetaan middleware, joka ylikirjoittaa nämä säädöt aina, jolloin cache-control olisi aina no-store